src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/sql.js

```diff
@@ -0,0 +1,274 @@
+loadScript('https://cdnjs.cloudflare.com/ajax/libs/sql-formatter/4.0.2/sql-formatter.min.js')
+
+class SqlVisualization extends Visualization {
+    static inputType = 'Any' // 'Standard.Database.Data.Table.Table | Standard.Database.Data.Column.Column' // TODO change this once sum types are properly supported
+    static label = 'SQL Query'
+
+    constructor(api) {
+        super(api)
+        this.setPreprocessorModule('Standard.Visualization.Sql.Visualization')
+        this.setPreprocessorCode(`x -> here.prepare_visualization x`)
+
+        // mock theme
```

Comment:
comments should be valid sentences. Also, they should be understandable by people not familiar with codebase, thisdefinitely is not. It needs to describe why it is a mock and what issue it is waiting for with a link to the issue.

Reply:
This mock theme will be removed anyway before the PR is merged, as we discussed that this PR should be merged after #1337